gwtp-core/gwtp-dispatch-rpc-client/src/main/java/com/gwtplatform/dispatch/rpc/client/gin/RpcDispatchAsyncModule.java

```diff
@@ -131,13 +131,17 @@ protected void configureDispatch() {
 
         bind(RpcDispatchCallFactory.class).to(DefaultRpcDispatchCallFactory.class).in(Singleton.class);
 
-        bind(DispatchAsync.class).to(RpcDispatchAsync.class).in(Singleton.class);
+        bind(DispatchAsync.class).to(getDispatchAsyncClass()).in(Singleton.class);
```

Comment:
Let`s make all bindings configurable. No arbitrary rules. Do you think you can wrap it up before releasing 1.4?